src/collections/_documentation/platforms/cocoa/index.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Cocoa
+---
+
+{% capture __alert_content -%}
+Version `5.0.0` is still in beta.
+{%- endcapture -%}
+{%- include components/alert.html
+  title="Note"
+  content=__alert_content
+  level="info"
+%}
+
+This is the documentation for our Cocoa SDK (Swift and Objective-C).
+If you are migrating from an older version, please consider our [Migration Guide](https://github.com/getsentry/sentry-cocoa/blob/master/MIGRATION.md). Also always make sure to follow the [changelog](https://github.com/getsentry/sentry-cocoa/blob/master/CHANGELOG.md)
+
+<!-- WIZARD -->
+## Installation {#install}
+
+The SDK can be installed using [CocoaPods](http://cocoapods.org), [Carthage](https://github.com/Carthage/Carthage), or [Swift Package Manager](https://swift.org/package-manager/). This is the recommended SDK for both Swift and Objective-C projects.
+
+It supports iOS (>= 8.0), tvOS (>= 9.0), macOS (>= 10.10) and partially watchOS (>= 2.0). 
+
+### CocoaPods
+
+Installing the SDK with CocoaPods is currently our recommended way.
```

Comment:
Suggest: "We recommend installing the SDK with CocoaPods." because "currently" dates very quickly.